mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -674,6 +674,14 @@ def _df_to_embeddings(self, df: pd.DataFrame) -> pd.DataFrame:
 
         model_id = self._kb.embedding_model_id
 
+        if model_id is None:
+            # call litellm handler
+            messages = list(df[TableField.CONTENT.value])
+            args = self._kb.params['embedding_model']
```

Comment:
The `default_embedding_model` should be used here if the `embedding_model` is not provided.

Reply:
thanks! 
updated